winpr/include/winpr/crt.h

```diff
@@ -169,11 +159,69 @@ extern "C"
 	WINPR_API errno_t memmove_s(void* dest, size_t numberOfElements, const void* src, size_t count);
 	WINPR_API errno_t wmemmove_s(WCHAR* dest, size_t numberOfElements, const WCHAR* src,
 	                             size_t count);
-
 #ifdef __cplusplus
 }
 #endif
 
 #endif /* _WIN32 */
 
+#if !defined(_WIN32) || (defined(__MINGW32__) && ! defined(_UCRT))
+/* note: we use our own implementation of _aligned_XXX function when:
+ * - it's not win32
+ * - it's mingw with native libs (not ucrt64) because we didn't managed to have it working
+ *   and not have C runtime deadly mixes
+ */
+#if WINPR_MSVCR_ALIGNMENT_EMULATE
```

Comment:
Leftovers? This does not make any sense in this branch as we do not use the `_aligned_malloc` et al in `FreeRDP` and `WinPR` any more.

Reply:
If you want to have this so that we export the old symbols, I´d suggest that you add a `CMake option` for `WINPR_MSVCR_ALIGNMENT_EMULATE`

Reply:
Well I've added these if someone still wants to use WinPR to compile win32 apps under linux

Reply:
ah, ok. but shouldn´t `#ifdef` or `#if defined()` be used instead? this way you need to do `#define WINPR_MSVCR_ALIGNMENT_EMULATE 1` instead of only `#define WINPR_MSVCR_ALIGNMENT_EMULATE`

Reply:
oh yes good catch